kubernetes/kubernetes_service.go

```diff
@@ -106,3 +123,27 @@ func filterByService(serviceName string, dl *v1beta1.DeploymentList) *v1beta1.De
 	dl.Items = deployments
 	return dl
 }
+
+func filterAutoscalersByDeployments(deploymentNames []string, al *autoscalingV1.HorizontalPodAutoscalerList) *autoscalingV1.HorizontalPodAutoscalerList {
+	var autoscalers []autoscalingV1.HorizontalPodAutoscaler
```

Comment:
I'm just learning go, but I guess slices are preferable rather than arrays for append operations.
Having said that I don't have a strong view if what is the gain/loss versus one or other.
I use slices :-) when using append.

Reply:
I saw this use in more areas of the code. Just leaving here a note to not add unnecessary noise.

Reply:
You are right, gotcha!